Formula/i2pd.rb

```diff
@@ -69,3 +71,77 @@ def post_install
     Process.kill "TERM", pid
   end
 end
+
+__END__
+diff --git a/libi2pd/RouterInfo.cpp b/libi2pd/RouterInfo.cpp
+index 8d6c7120..ee7acd73 100644
+--- a/libi2pd/RouterInfo.cpp
++++ b/libi2pd/RouterInfo.cpp
+@@ -46,7 +46,7 @@ namespace data
```

Comment:
Is this submitted upstream?

Reply:
Yes, read commit description - I provided commit hash in upstream.

Reply:
Can't we apply that commit directly?

Reply:
It was created ahead of trunk code, which contains changes in modified files.

Reply:
If it's just a warning it's fine, a commit reference is much easier to keep track of than an inline patch.